src/OrchardCore.Modules/OrchardCore.Tenants/Views/Admin/Index.cshtml

```diff
@@ -16,6 +17,13 @@
     {
         var shellSettings = shellSettingsEntry.ShellSettings;
         var host = shellSettings.RequestUrlHost?.Split(new[] { ',', ' ' }, StringSplitOptions.RemoveEmptyEntries).FirstOrDefault();
+        
+        if (host != null)
```

Comment:
This should basically never happen but there is another issue making this to happen.